pkg/features/versioned_kube_features.go

```diff
@@ -28,11 +28,407 @@ import (
 // Entries are separated from each other with blank lines to avoid sweeping gofmt changes
 // when adding or removing one entry.
 var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate.VersionedSpecs{
+	CrossNamespaceVolumeDataSource: {
+		{Version: version.MustParse("1.26"), Default: false, PreRelease: featuregate.Alpha},
+	},
+	AnyVolumeDataSource: {
+		{Version: version.MustParse("1.18"), Default: false, PreRelease: featuregate.Alpha},
+		{Version: version.MustParse("1.24"), Default: true, PreRelease: featuregate.Beta},
+	},
+	AppArmor: {
+		{Version: version.MustParse("1.4"), Default: true, PreRelease: featuregate.Beta},
+		{Version: version.MustParse("1.31"), Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.33
+	},
+	AppArmorFields: {
+		{Version: version.MustParse("1.30"), Default: true, PreRelease: featuregate.Beta},
+		{Version: version.MustParse("1.31"), Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.33
+	},
+	AuthorizeNodeWithSelectors: {
+		{Version: version.MustParse("1.31"), Default: false, PreRelease: featuregate.Alpha},
+	},
+	ClusterTrustBundle: {
+		{Version: version.MustParse("1.26"), Default: false, PreRelease: featuregate.Alpha},
+	},
+	ClusterTrustBundleProjection: {
+		{Version: version.MustParse("1.28"), Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Website says 1.29: https://github.com/kubernetes/website/blob/1c5221a29971e1658eff8f116dbb4ffa0605927b/content/en/docs/reference/command-line-tools-reference/feature-gates/cluster-trust-bundle-projection.md?plain=1#L11

which is correct?

Reply:
It should be v1.28, the website is wrong IIUC. This PR has ClusterTrustBundleProjection alpha: 1.28 which is the first time it was introduced and added to kube_features.go:
https://github.com/kubernetes/kubernetes/pull/113374/files#diff-71e3b98f9a6bbf5b8421e26a7ba0c079f397cd8d49abacdad943c66a4f44f03d

![image](https://github.com/user-attachments/assets/94282337-2290-4afd-b5f3-e9e6d825b1fd)

Reply:
In investigating this more I believe the website is correct. Although this feature has a PR adding it in alpha @ v1.28 in looking at kube_features.go file for the v1.28 release (link below for v1.28.13) this feature isn't actually present. I believe the website is correct and the current kube_features.go files is incorrect

`Ctrl-F` `ClusterTrustBundleProjection` on the v1.28 release kube_features.go file below, the feature isn't present // in alpha
https://github.com/kubernetes/kubernetes/blob/v1.28.13/pkg/features/kube_features.go

Reply:
Thanks for following up.

Reply:
Fixed to 1.29